Firestore/Protos/build_protos.py

```diff
@@ -119,7 +122,13 @@ def __run_generator(self, out_dir):
 
     cmd.extend(self.proto_files)
 
-    subprocess.check_call(cmd)
+    kwargs={}
+    if self.args.protoc_pythonpath:
+      env = os.environ.copy()
+      env['PYTHONPATH'] = self.args.protoc_pythonpath
```

Comment:
This effectively overrides the user's PYTHONPATH, which could cause other things to break. Prepending to the existing path rather than overwriting might be better.

Reply:
Done.